src/doc/src/guide/creating-a-new-workspace.md

```diff
@@ -0,0 +1,141 @@
+# Creating a New Workspace
+
+A [workspace][def-workspace] is a collection of one or more packages, 
+called workspace members, that are managed together.
+
+In this chapter, we will create a workspace `new_workspace` containing 
+binary member `foo` and library member `bar`.
+
+As mentioned in [`[workspace]` section][workspace-section], the workspace must 
+have at least one member, either the [root package] or a [virtual manifest].
+
+Next we create a workspace containing [root package].
+For convenience, you can first create a package using the command `cargo new new_workspace`.
+Then add the `[workspace]` section to the `Cargo.toml` file in the root directory 
+to make it a manifest of the workspace:
+
+```toml
+# [new_workspace]/Cargo.toml
+[workspace]
+
+[package]
+name = "new_workspace"
+version = "0.1.0"
+edition = "2021"
```

Comment:
This feels odd because `new_workspace` is actually a package. The step-by-step guide is not an ideal workflow, and even more confusing at this moment. Admittedly this is the tool's failure <https://github.com/rust-lang/cargo/issues/8365>. I wonder if we want to block this on that.